modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -67,11 +67,23 @@ def initialize(user, options = {})
         @user = user
         fetch_additional_resources
         resource = UserStruct.new(user.uuid, profile, authorized_services, health)
+        province_log(profile)
         super(resource, options)
       end
 
       private
 
+      # No domestic or military addresses should have a province but some have been coming in as a string 'null'
```

Comment:
In general, I would rather we do this in the controller because it's definitely not part of serialization, but I'm fine with this as a temporary measure. Let's just be sure to remove this soon. We should remove the suggested address logging soon as well.